Remove PATTERNLINT suppressions

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #6693
* __->__ #6692

The linter these are suppressing was removed in D63859186.

Differential Revision: [D65507718](https://our.internmc.facebook.com/intern/diff/D65507718/)

swolchok committed:
[ExecuTorch] Remove PATTERNLINT suppressions

The linter these are suppressing was removed in D63859186.

Differential Revision: [D65507718](https://our.internmc.facebook.com/intern/diff/D65507718/)

[ghstack-poisoned]

kimishpatel reviewed (changes requested):
you might have to reoroder the stack. I see linter ci job failing due to the fixes in the pr

swolchok committed:
Update on "Remove PATTERNLINT suppressions"


The linter these are suppressing was removed in D63859186.

Differential Revision: [D65507718](https://our.internmc.facebook.com/intern/diff/D65507718/)

[ghstack-poisoned]

swolchok committed:
Update on "Remove PATTERNLINT suppressions"


The linter these are suppressing was removed in D63859186.

Differential Revision: [D65507718](https://our.internmc.facebook.com/intern/diff/D65507718/)

[ghstack-poisoned]
